scrooge-sbt-plugin/src/sbt-test/scrooge/basic/build.sbt

```diff
@@ -0,0 +1,10 @@
+
+scalaVersion := "2.11.8"
+
+enablePlugins(BuildInfoPlugin)
+
+libraryDependencies ++= Seq(
```

Comment:
Yea I was hoping for some direction on what to do with these. I thought it might be ok if the plugin is backwards compatible, but if that is not the case, we could export some more props for these versions.d